packages/realm/src/platform/node/device-info.ts

```diff
@@ -41,7 +40,7 @@ inject({
       frameworkName: typeof process.versions.electron === "string" ? "Electron" : "Node.js",
       frameworkVersion: process.versions.electron || process.version,
 
-      bundleId: REALM_ANONYMIZED_BUNDLE_ID,
+      bundleId: config?.anonymizedBundleId || "unknown",
```

Comment:
Nit: Looked like the `config` was set to an object in another file if there was no `config`, unsure if it's the same path, but is `?` needed here? (Comment applies to `/react-native/device-info.ts` as well)